src/Gateway/PayplugGatewayOney3x.php

```diff
@@ -125,8 +125,9 @@ public function check_oney_is_available()
         }
 
         // Country check
-        $country_code = WC()->customer->get_shipping_country();
-        if (!in_array($country_code, $this->allowed_country_codes)) {
+        $country_code_shipping = WC()->customer->get_shipping_country();
+        $country_code_billing = WC()->customer->get_billing_country();
+        if (!in_array($country_code_billing, $this->allowed_country_codes) && !in_array($country_code_shipping, $this->allowed_country_codes)) {
```

Comment:
```suggestion
        if (!in_array($country_code_billing, $this->allowed_country_codes) || !in_array($country_code_shipping, $this->allowed_country_codes)) {
```

C'est pas un OR plutôt ?